web/src/settings_components.js

```diff
@@ -681,6 +681,23 @@ function enable_or_disable_save_button($subsection_elem) {
         disable_save_btn = should_disable_save_button_for_time_limit_settings(time_limit_settings);
     } else if ($subsection_elem.attr("id") === "org-other-settings") {
         disable_save_btn = should_disable_save_button_for_jitsi_server_url_setting();
+        const $button_wrapper = $subsection_elem.find(".subsection-changes-save");
+        const is_tippy = $button_wrapper.get(0)._tippy;
+        if (disable_save_btn) {
+            if (!is_tippy) {
```

Comment:
Do we need this `is_tippy` condition? I don't think we do this check at other places.

Reply:
We added a clarifying comment that explains why we need this. In short, we need to check that there is not already a tippy in place since this function runs every time a letter is entered in the URL box.